### PR TITLE
Upgrade mongodb drivers to 1.5.3 and PHP version

### PR DESCRIPTION
**Pull request by erickhun**
- Use the last mongodb driver `1.5.3`, compatible with PHP `5.6` and `7.1` 
- Upgrade PHP minor version from `7.1.25` to 17.1.27`, [only bugfixes improvements](http://www.php.net/ChangeLog-7.php#7.1.27) 
- freeze `igbinary` and `imagick` versions
- add `sockets` extensions (that was set in children image)
- made the tag convention more simple with the php version only instead of a lengthy thing

@colinscape up for a review ? :)

### CODE REVIEW

File: apache-php/7.1/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 IMG_NAME := bufferapp/apache-php
-IMG_VERSION := 7.1.25-apache-stretch-igbinary-mongodb-1.3.4
+IMG_VERSION := 7.1.27
```
Comment (colinscape) [inline apache-php/7.1/Makefile]: So clean! 🤩

File: apache-php/5.6/Dockerfile [hunk 2]
```diff
@@ -18,11 +18,11 @@ RUN apt-get update && \
 
 # Drivers and libraries
 RUN docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/
-RUN docker-php-ext-install -j$(nproc) gd mcrypt calendar pcntl
-RUN pecl install -f mongodb-1.3.4 && \
+RUN docker-php-ext-install -j$(nproc) gd mcrypt calendar pcntl sockets
```
Comment (colinscape) [inline apache-php/5.6/Dockerfile]: Curious about the addition of sockets here (and below). Was this something we were missing before (I guess for the workers and their graceful terminations?)

Comment (erickhun) [reply apache-php/5.6/Dockerfile]: @colinscape I think we need that for the datadog library to send the metrics to the dd-agent

Comment (colinscape) [reply apache-php/5.6/Dockerfile]: Ah! Makes sense! 👍

**Pull request merged**